CMPUT404Fall2021-6803d618/social.distance
# Implement sending a FriendRequest.

new endpoint:
- `/author/<author_id>/friend_request/<foreign_author_url>/`
- `/author/<aid>/inbox/`

testing:
1. create two authors, say A and B
2. run two instances of the server: `./manage.py runserver 8000` and `./manage.py runserver 8001`, we test on 8000.
3. copy the url for B, except change its url from localhost:8000/.../ to localhost:8001/.../
2. on 8000 server, POST to `/author/<A_id>/friend_request/<B_url>/`, we expect 8000 to POST something to 8001, where B_url points to.
3. on either server, GET `/author/<B_id>/inbox/` and check if the inbox has a FriendRequest item.
